### PR TITLE
Be search multiple page

### PR DESCRIPTION
**Pull request by cpfergus1**
- [x] Updated Files:

- [x] Routes Added:

- [x] Repo Representation: Front End, Back End, Amedaus, Open Weather, Google 

- [ ] Sad Path Tested - NEED SAD PATH TESTING, Generated new card (search sad path, params sad path, etc)

- [x] Happy Path Tested

- [x] All Tests are passing

- [ ] Updated README.md with added information

- [x] Collabotors: Sheryl

### CODE REVIEW

File: app/controllers/flights_controller.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    #   f.params[:trip_duration] = flight_params[:trip_duration]
+    # end
+    # json = JSON.parse(response.body, symbolize_names: true)
+  end
+
+  private
+  def flight_params
+    params.permit(:departure_airport, :departure_date, :trip_duration)
+  end
+end
```
Comment (JesseMellinger) [inline app/controllers/flights_controller.rb:30]: Like seeing the use of strong params here! Nice work.

File: app/poros/trip.rb [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+             :trip_duration,
+             :max_f,
+             :min_f,
+             :max_c,
+             :min_c,
+             :day_feels_like_c,
+             :day_feels_like_f,
+             :sky_coverage,
+             :booking_link,
+             :weather_data_available,
```
Comment (JesseMellinger) [inline app/poros/trip.rb:19]: I think we can safely remove the `weather_data_available` attribute for now.

File: app/services/backend_service.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
+class BackendService
+  def self.conn
+    Faraday.new(url: ENV["BACKEND_URL"])
+  end
+
+  def self.flights_search(query)
+    response = conn.get('/flight_search') do |req|
```
Comment (JesseMellinger) [inline app/services/backend_service.rb:7]: Currently our search route in BE is `/api/v1/search` so I think we'll want to change the `/flight_search' path here.

**Review by JesseMellinger — CHANGES_REQUESTED** [review]
Left a few comments to review. Nice work here!

File: spec/fixtures/flight_data_return.json [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+				            "day_feels_like_c":-7.0499999999999545,
+				            "day_feels_like_f":24.950000000000045,
+				            "description":"clear sky",
+				            "date":"2021-01-09T02:00:00+00:00",
+				            "chance_precip":20,
+				            "sky_coverage":20,
+				            "weather_data_available":true
+				          }
+				        }
+    ]
```
Comment (GeorgieGirl24) [inline spec/fixtures/flight_data_return.json:168]: Nice on this fixture file! I think that the `weather_data_available` can be removed.

**Review by GeorgieGirl24 — COMMENTED** [review]
Nice work here! You are making great progress!

File: app/views/flights/index.html.erb [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+    <th>Price</th>
+    <th>Trip Duration</th>
+    <th>Weather</th>
+  </tr>
+  <% @flights.each do |flight| %>
+  <tr class="Flight" id="flight-<%=flight.flight_id%>">
+    <td class="DestinationCity"><%= link_to flight.destination_city %></td>
+    <td class="Price"><%= flight.price %></td>
+    <td class="Duration"><%= flight.trip_duration %></td>
+    <td class="Weather">Hight: <%= flight.max_f.round(1) %>, <br>
```
Comment (nmking22) [inline app/views/flights/index.html.erb:14]: Looks like a typo here Hight -> High

**Review by nmking22 — APPROVED** [review]
Awesome job -  I love the use of the table in the view, and the within block testing! I left one note about a typo, but this is fantastic!

File: spec/fixtures/flight_data_return.json (hunk 4 above)
Comment (dunlapww) [reply spec/fixtures/flight_data_return.json:168]: Awesome work on this.  I realized some typos on the weather api readme (i made) - we are not currently we're not passing back either 'chance_precip' or 'sky_coverage' from the weather api.  I'm not sure if those are even fields in the api.

File: app/views/flights/index.html.erb (hunk 5 above)
Comment (stillsheryl) [reply app/views/flights/index.html.erb]: thanks, fixed now!

File: app/poros/trip.rb (hunk 2 above)
Comment (stillsheryl) [reply app/poros/trip.rb]: removed, thanks!

File: spec/fixtures/flight_data_return.json (hunk 4 above)
Comment (stillsheryl) [reply spec/fixtures/flight_data_return.json:163]: This has been updated. Thanks!

File: app/services/backend_service.rb (hunk 3 above)
Comment (stillsheryl) [reply app/services/backend_service.rb]: This has been updated.

**Review by JesseMellinger — APPROVED** [review]

**Review by GeorgieGirl24 — APPROVED** [review]

**Pull request merged**